source/hardware/controllers/hercules_djcontrol_inpulse_500.rst

```diff
@@ -0,0 +1,316 @@
+Hercules DJControl Inpulse 500
+==============================
+
+.. figure:: ../../_static/controllers/hercules_djcontrol_inpulse_500.svg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Hercules DJControl Inpulse 500 (schematic view)
+   :figclass: pretty-figures
+
+   Hercules DJControl Inpulse 500 (schematic view)
+
+-  `Manufacturer’s product page <https://web.archive.org/web/20241013231548/https://www.hercules.com/en-us/product/djcontrol-inpulse-500/>`__
+-  `Manufacturer’s support and downloads page <https://support.hercules.com/en/product/djcontrolinpulse500-en//>`__
+-  `Forum thread <https://mixxx.discourse.group/t/hercules-djcontrol-inpulse-500/19739/101/>`__
+
+.. versionadded:: 2.6.0
+
+Compatibility
+-------------
+
+This controller is a class compliant USB MIDI and audio device, so it
+can be used without any special drivers on GNU/Linux, Mac OS X, and
+Windows. However, if you wish to use the :ref:`ASIO sound API <preferences-sound-api>`
+under Windows, please install the latest driver package available from
+the `Support
+page <https://support.hercules.com/en/product/djcontrolinpulse500-en//>`__.
+
+Sound card setup
+----------------
+
+This controller has built-in 4 channel output sound card, with MAIN output
+(2x RCA or 2x unbalanced 6.35mm jack) and HEADPHONE output (stereo
+3.5mm jack or stereo 6.35mm jack)
+
+-  Open **Preferences > Sound Hardware**
+-  Select the **Output** tab
+-  From the **Main** drop-down menu, select the audio interface, then
+   **Channels 1-2**
+-  From the **Headphones** drop-down menu, select the audio interface,
+   then **Channels 3-4**
+-  Click **Apply** to save the changes.
+
+.. seealso::
+   The :ref:`example setups section <setup-laptop-and-external-card>` provides more details about the audio configuration in Mixxx.
+
+.. note::
+   The **Master** and **Headphone** knobs, as well as the
+   **Master** button are hardware controls and interact directly with the
+   integrated sound card’s output. Although they also send MIDI messages,
+   they have NOT been mapped in Mixxx, so do not expect an on-screen
+   reaction when using them. This was done to prevent the knobs to adjust
+   both the gain on the controller’s sound card and in Mixxx.
+
+Mapping description
+-------------------
+
+Decks
+~~~~~
+
+===============================================================  ==========================================
+Control                                                          Function
+===============================================================  ==========================================
+:hwlabel:`SYNC` button                                           Toggle :ref:`Sync Lock <sync-lock>`.
+:hwlabel:`SHIFT` + :hwlabel:`SYNC` button                        Match key to currently playing track.
+:hwlabel:`CUE` button                                            Specifies, plays or recalls temporary cue point.
```

Comment:
If this is mapped to `cue_default` we should link to [Using Cue Modes](https://manual.mixxx.org/latest/chapters/user_interface.html#using-cue-modes)

Reply:
It is, right, @resetreboot  ?